src/microbe_stage/compound_cloud_system.cpp

```diff
@@ -1178,7 +1178,7 @@ void
             // TODO: move this to the shaders for better performance (we would
             // need to pass a float instead of a byte).
             int intensity =
-                static_cast<int>(255 * std::log(density[i][j] / 1000 + 1));
+                static_cast<int>(255 * 2 * std::atan(0.003 * density[i][j]));
```

Comment:
Does 0.003 here have to a double. It might perform better if it was a float as I think the density is a float, so with just two floats being multiplied it wouldn't cause 64 bit multiply, which might be slower?